src/ChildBridge.js

```diff
@@ -27,7 +27,11 @@ class ChildBridge extends React.Component {
 
   events(child){
     let { events, onEvent } = this.props;
-    events = events == null ? [] : [].concat(events);
+    if (events == null) {
```

Comment:
(I can't actually test this because this is apparently many versions ahead of the dep for react-formal/react-input-message)